system-setup.py

```diff
@@ -20,31 +20,30 @@ def common_first(self):
         self.pip_install("setuptools --upgrade")
 
         self.install("git jq curl")
-        self.run("%s/bin/getgcc" % READIES)
+        self.run("%s/bin/enable-utf8" % READIES)
 
     def debian_compat(self):
-        pass
+        self.run("%s/bin/getgcc" % READIES)
```

Comment:
@danni-m We're now serving each platform with the gcc it deserves (mainly because centos needs `--modern` and macOS comes with its own clang installed).